README.md

```diff
@@ -16,10 +16,10 @@ Further development of the EF6 PowerTools is happening in a community-driven pro
 
 If you want to use an officially supported Entity Framework release to develop your applications then head to https://docs.microsoft.com/ef/ef6/ where you can find installation information, documentation, tutorials, samples, and videos.
 
-If you want to try out the latest changes that have not been officially released yet, you can choose to [build the code](https://github.com/aspnet/EntityFramework6/wiki/Building-the-Runtime). We regularily also make [nightly builds](https://github.com/aspnet/EntityFramework6/wiki/Nightly-Builds) of the Entity Framework codebase available.
+If you want to try out the latest changes that have not been officially released yet, you can choose to [build the code](https://github.com/dotnet/ef6/wiki/Building-the-Runtime). We regularily also make [nightly builds](https://github.com/dotnet/ef6/wiki/Nightly-Builds) of the Entity Framework codebase available.
 
 ## How do I contribute
 
-There are lots of ways to [contribute to the Entity Framework project](https://github.com/aspnet/EntityFramework6/wiki/Contributing) including [testing out nighty builds](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
+There are lots of ways to [contribute to the Entity Framework project](https://github.com/dotnet/ef6/wiki/Contributing) including [testing out nighty builds](https://github.com//blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
```

Comment:
The replacement for aspnet/AspNetCore looks like it's missing?

Reply:
```suggestion
There are lots of ways to [contribute to the Entity Framework project](https://github.com/dotnet/ef6/wiki/Contributing) including [testing out nighty builds](https://github.com/dotnet/aspnetcore/blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
```